src/main/kotlin/no/nb/mlt/wls/order/controller/OrderController.kt

```diff
@@ -72,4 +76,41 @@ class OrderController(val orderService: OrderService) {
     suspend fun createOrder(
         @RequestBody payload: ApiOrderPayload
     ): ResponseEntity<ApiOrderPayload> = orderService.createOrder(payload)
+
+    @Operation(
+        summary = "Gets an order from the storage system",
+        description = "Checks if a specified order exists within Hermes WLS."
```

Comment:
Should probably expand on the API doc, but since I like doing it, I can fix it later on